system_tests/features/get_info_endpoints.feature

```diff
@@ -1,16 +1,25 @@
 Feature: Get info endpoints
 
   Scenario Outline: System hits service get info endpoint
-    Given the system requests <service> endpoint info
-    When  the <service> endpoint info is returned
+    Given the <service> service is running
```

Comment:
Suggest we change the `Given` statement to `Given services are running`. This makes it common to all examples. If we want to separate them, then we might need to use step matchers e.g. https://jenisys.github.io/behave.example/step_matcher/using_matchers.html